rero_ils/modules/collections/api.py

```diff
@@ -88,6 +89,17 @@ class CollectionsIndexer(IlsRecordsIndexer):
 
     record_cls = Collection
 
+    def index(self, record):
```

Comment:
The only problem here is that a simple update of the `collection` record triggers the indexing of all of its items `normally a lot` in the document record. There are many fields in the `collection` records that does not need an indexing at the document level (`subjects` for example).

Reply:
@BadrAly I'm agree with you. Do I detect change only on title and description fields?
In this case, the only solution that I know is to create a listener on `before_record_update`. But this cause a call to this method for each update of any ressources (first operation of this listener should be to detect if the update is about a Collection).
What's you opinion about this solution ?